src/index.html

```diff
@@ -1,19 +1,597 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Title</title>
+
+    <title>NAMU</title>
+
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="/src/images/icons/fav-icon.png"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/main.scss"
     />
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page__body">
+    <div class="header">
+      <div class="container container--header">
+        <div class="top-bar header__top-bar">
+          <a href="#">
+            <img
+              src="./images/logo.svg"
+              alt="NAMU logo"
+              class="top-bar__logo"
+            />
+          </a>
+
+          <div class="top-bar__icons">
+            <a
+              href="#menu"
+              class="icon icon--menu top-bar__icon"
+            ></a>
+
+            <div class="language language--header top-bar__language">
+              <select
+                required="required"
+                class="language__select"
+              >
+                <option
+                  class="language__option"
+                  value="ua"
+                >
+                  UA
+                </option>
+              </select>
+            </div>
+          </div>
+        </div>
+
+        <div class="header__main-content">
+          <p class="header__date">10 серпня - 10 листопада</p>
+
+          <h1 class="header__title">Мистецтво ХІХ - ХХ ст.</h1>
+
+          <p class="header__subtitle">
+            Внесок українських митців у світову культуру 19-20ст
+          </p>
+
+          <a
+            href="#plan-visit"
+            class="primary-button header__button"
+          >
+            Купити квиток
+          </a>
```

Comment:
`<a>` tags used as buttons should include `role="button"` to ensure better screen reader support.